trdg/utils.py

```diff
@@ -145,4 +145,4 @@ def get_text_height(image_font: ImageFont, text: str) -> int:
     """
     Get the width of a string when rendered with a given font
     """
-    return image_font.getsize(text)[1]
+    return image_font.getbbox(text)[1]
```

Comment:
This usage of `getbbox` is incorrect. `getsize` returns a `(width, height)` tuple. `getbbox` returns a `(left, top, right, bottom)` tuple. See https://github.com/Belval/TextRecognitionDataGenerator/pull/323.